cluster-ha-singleton/README.md

```diff
@@ -76,16 +76,16 @@ Build and Deploy the Quickstart
 2. Open a command prompt and navigate to the root directory of this quickstart.
 3. Type this command to build and deploy the archive:
 
-        mvn clean install jboss-as:deploy
+        mvn clean install wildfly:deploy
 
 4. This deploys `service/target/jboss-cluster-ha-singleton-service.jar` to the running instance of the first server.
 5. Since default socket binding port is `9999` and the second server runs at a port offset of `100`, you must pass port `10099` (9999 + 100) as an argument when you deploy to the second server. Type this command to deploy the archive to the second server. 
 
-        mvn jboss-as:deploy -Djboss-as.port=10099
+        mvn wildfly:deploy -Djboss-as.port=10099
     
     If the second server is on a different host, you must also pass an argument for the host name as follows:
     
-        mvn jboss-as:deploy [-Djboss-as.hostname=OTHERHOST] -Djboss-as.port=10099
```

Comment:
The property have changed from jboss-as.port to wildfly.port. See: http://docs.jboss.org/wildfly/plugins/maven/latest/deploy-mojo.html#port